Flank/flank
# feat: Change uploading file output

Fixes #1334 

Previous uploading Flank output looks like:
```
Saved 1 shards to android_shards.json
  Uploading android_shards.json .
  Uploading app-debug.apk .
  Uploading app-debug-androidTest.apk .
  1 test / 1 shard

  1 matrix ids created in 0m 3s
  https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir

```

The new output for file upload looks more like Gcloud output:
```
  Saved 1 shards to android_shards.json
  Uploading [android_shards.json] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir/...
  Uploading [app-debug.apk] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir/...
  Uploading [app-debug-androidTest.apk] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir/...

  1 test / 1 shard

  1 matrix ids created in 0m 3s
  Raw results will be stored in your GCS bucket at [https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir]
```


Gcloud file upload output for reference:
```
Uploading [test_runner/../test_projects/android/apks/app-debug.apk] to Firebase Test Lab...
Uploading [test_runner/../test_projects/android/apks/app-debug-androidTest.apk] to Firebase Test Lab...
Raw results will be stored in your GCS bucket at [https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/test_dir/]

```

## Test Plan
> How do we know the code works?

1. Run Flank
1. Check output
1. Output for file upload looks like described above


## Checklist

- [x] Research done
- [x] Some refactor of output
